src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -223,6 +225,9 @@
     //  "TenantId": "",
     //  "CallbackPath": "/signin-oidc",
     //  "SaveTokens": false
+    //},
+    //"OrchardCore_Tenants": {
+    //  "TenantRemovalAllowed": true // Whether tenant removal is allowed, false by default.
```

Comment:
Mention the removal feature on the Tenants docs page, and this config too.

Reply:
Okay done